Lib/test/test_httplib.py

```diff
@@ -1082,7 +1082,7 @@ def test_chunked(self):
         resp.close()
 
         # Various read sizes
-        for n in range(1, 12):
+        for n in list(range(1, 12)) + [-1, None]:
```

Comment:
I would prefer a different test, namely:

```py
# explicit full read
for n in (-1, None):
    with self.subTest('full read', n=n):
        sock = FakeSocket(chunked_start + last_chunk + chunked_end)
        resp = client.HTTPResponse(sock, method="GET")
        resp.begin()
        self.assertEqual(resp.read(n), expected)
        resp.close()
```

We want to check that `-1` and `None` read everything, right?

Reply:
Yep, thanks for help!

Reply:
(Btw, you need to sign the CLA once again with your new account)

Reply:
Hum. I don't know how to fix this so the best way would be to close this PR and open a new one (apparently the CLA is still not consider to be signed).

Reply:
I need help here.
I made several commits with difference emails, manushkin@gmail.com and manushkin@nebius.com
The first one is my own email, and I signed the CLA using it.
But the second is my job email, and I don't understand how to sign the CLA with it.

Is it possible to remove second email?

Reply:
Errr... ok, let's pause. First, locate your first commit:

```sh
$ git rebase -i 974f9287b84ecc83b2765a208c144b4bc25197b8~1
```

This will open an interactive editor. You will have something like:

```
pick 974f92 ...
pick ...
pick ...
```

After each commit write:

```sh
exec "git commit --amend --author='YOUR_NAME <YOUR_EMAIL_YOU_SIGNED_THE_CLA_WITH>' -C HEAD"
```

This will looks like:

```sh
pick 974f92
exec "git commit --amend --author='YOUR_NAME <YOUR_EMAIL_YOU_SIGNED_THE_CLA_WITH>' -C HEAD"
pick ....
exec "git commit --amend --author='YOUR_NAME <YOUR_EMAIL_YOU_SIGNED_THE_CLA_WITH>' -C HEAD"
pick ...
exec "git commit --amend --author='YOUR_NAME <YOUR_EMAIL_YOU_SIGNED_THE_CLA_WITH>' -C HEAD"
...
```

Close the editor. Run `git log` and check that the emails have been updated accoringly. It *should* now be fine. Then do `git push -f` (we need to force-push this). Hopefully, this will work. If it doesn't, we'll just create a new PR where you carefully set the author's email everytime.

Reply:
Thanks, it helped!